### PR TITLE
added sources below preview

### PR DESCRIPTION
**Pull request by Epsilon02**

### CODE REVIEW

**Comment by 6543** [comment]
Bildschirmfoto zu 2021-03-13 16-46-10
 ??

**Comment by Epsilon02** [comment]
Is your Programm in english?

File: src/app/preview/preview.component.ts [hunk 1]
```diff
@@ -46,22 +48,29 @@ export class PreviewComponent implements OnInit {
...
-            hPhrases: value.hPhrases.modifiedData ?? value.hPhrases.originalData,
-            pPhrases: value.pPhrases.modifiedData ?? value.pPhrases.originalData,
-            signalWord: value.signalWord.modifiedData ?? value.signalWord.originalData,
-            symbols: value.symbols.modifiedData ?? value.symbols.originalData,
-            lethalDose: value.lethalDose.modifiedData ?? value.lethalDose.originalData,
-            mak: value.mak.modifiedData ?? value.mak.originalData,
-            amount: value.amount.modifiedData ?? value.amount.originalData,
-          })),
+          data.map<SimpleSubstanceData>((value) => {
+            console.log(this.sources.indexOf(value.source.provider));
```
Comment (Epsilon02) [inline src/app/preview/preview.component.ts:52]:
```suggestion
```

**Comment by crapStone** [comment]
could you put it between substances and h-/p-phrases and in a single line?

**Review by crapStone — APPROVED** [review]

**Pull request merged**